### PR TITLE
Fix/400 dandelion sprouts annoyances

### PR DESCRIPTION
**Pull request by Alex-302**
**Do not merge.**
Build fails because `https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AnnoyancesList` ~has no extension~

### CODE REVIEW

**Comment by ameshkov** [comment]
> Build fails because https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AnnoyancesList has no extension

Nope, actually, it cannot handle on of inclusions:

`TypeError [ERR_UNESCAPED_CHARACTERS]: Request path contains unescaped characters`

**Comment by Alex-302** [comment]
Yep. Caused by these names
```
!#include stayingonbrowser/Staying%20On%20The%20Phone%20Browser
!#include Alternate%20versions%20Anti-Malware%20List/AntiMalwareAdGuard.txt
!#include Android%20Scum%20Class%20%E2%80%94%20Fake%20notification%20counters.txt
!#include Anti-'Custom%20cursors'%20List.txt
!#include I%20Don't%20Want%20to%20Download%20Your%20Browser.txt
!#include Dandelion%20Sprout's%20Anti-Malware%20List.txt
```

**Review by zloyden — APPROVED** [review]

**Comment by tvinzz** [comment]
updated FiltersCompiler and fixed

**Review by zloyden — APPROVED** [review]

File: filters/ThirdParty/filter_250_DandelionSproutAnnoyances/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "filterId": 250,
+  "name": "Dandelion Sprout's Annoyances List",
+  "description": "This list is made in the style of Fanboy's and AdGuard's annoyances lists, to show that I can make a just as good such annoyances list as them. It combines many of my proudest and most frequently maintained international lists, as a proof-of-concept and as a curated list for simplicity's sake.",
```
Comment (zloyden) [inline filters/ThirdParty/filter_250_DandelionSproutAnnoyances/metadata.json:4]: Btw, there is a new description https://github.com/AdguardTeam/FiltersRegistry/issues/400#issuecomment-792999470 without this part `, to show that I can make a just as good such annoyances list as them.`.
```
"This list is made in the style of AdGuard's and Fanboy's annoyances lists. It combines many of Dandelion Sprout's proudest and most frequently maintained international lists, as a curated compilation for simplicity's sake."
```

Comment (Alex-302) [reply filters/ThirdParty/filter_250_DandelionSproutAnnoyances/metadata.json]: Changed description

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**